bin/main

```diff
@@ -1,3 +1,60 @@
+# Variable definition area
 #!/usr/bin/env ruby
 
-puts 'Hello World!'
+board = %w[1 2 3 4 5 6 7 8 9]
+i = 0
+name_pone = nil
+name_ptwo = nil
+def display_board(board)
+  puts "#{board[0]} | #{board[1]} | #{board[2]}"
+  puts "---------"
+  puts "#{board[3]} | #{board[4]} | #{board[5]}"
+  puts "---------"
+  puts "#{board[6]} | #{board[7]} | #{board[8]} \n"
+end
+
+def input_request(num)
+  if num.even?
+    puts "it's #{@name_pone} turn"
+    board_input = gets.chomp
+  else
+    puts "it's #{@name_ptwo} turn"
+    board_input = gets.chomp
+  end
+end
+
+def welcome
+  puts "Welcome to Tic Tac Toe"
+  puts "what's player one's name ?"
+  @name_pone = gets.chomp
+  puts "what's player two's name ?"
+  @name_ptwo = gets.chomp
+  system('cls') || system('clear')
+end
+
+welcome
+
+while i < 9
+  given_number = input_request(i)
+  display_board(board)
```

Comment:
## Finish game
* `[REQUIRED]` game informs player if the selected move is a winning move
*  `[REQUIRED]` game informs player if the selected move is a draw move
- Take in mind the game finish when there's a winner or draw, not necessary the player should have 9 moves
  - [ ] Replace with another variable. Mockup all the logic with these variables
  *Example:*

  ```ruby
  while theresNoWinner && !isNotDraw
  # all the game logic
  end
  ```